test/files/mock-insights

```diff
@@ -13,47 +14,92 @@
 # auto_config=False
 # base_url=127.0.0.1:8888/r/insights
 # insecure_connection=True
+# username=admin
+# password=foobar
```

Comment:
Please add here: `gpg=False`. The insights-client does not work, when this configuration option is missing.

Reply:
It is there now.  A intermediate version of this PR had removed it by accident, but now it is back.